incubator/cassandra/sample/create-storage-gce.yaml

```diff
@@ -0,0 +1,7 @@
+kind: StorageClass
```

Comment:
This file is an example of how an administrator would create a storage class. It is not a part of kubernetes chart. I just put it in `sample` folder so the administrator will easily find out how to create a storage class. Or do you think shouldn't I keep this file?